ksp_compiler3/ksp_compiler_extras.py

```diff
@@ -505,10 +505,10 @@ def visitDeclareStmt(self, parent, node, *args):
             size = 1
 
         initial_value = None
-        if 'const' in node.modifiers:
+        if 'const' in node.modifiers and not ( isinstance(node.initial_value, Integer) or isinstance(node.initial_value, Real) ):
```

Comment:
Minor formatting nitpick - please remove the whitespace after opening and closing outer parens. Thanks!